### PR TITLE
modifications syntaxes + relations

### PR DESCRIPTION
**Pull request by likeAZ**
Modification de la relation entre status et customerCard + relation e…  …
…ntre comment et customerCard
f042322
 @likeAZ
likeAZ
Modification de la relation entre status et customerCard + relation e…  …
…ntre comment et customerCard
8dc21df
 @likeAZ
likeAZ
Merge branch 'master' of https://github.com/team-workshop-epsi-montpe…  …
…llier-b3/gfi-project-b3
fa71b80
 @likeAZ
likeAZ
changement d'une majuscule dans la variable card
0cd856f
 @likeAZ
likeAZ
modification relation entre comment et custommerCard
8701ea9
 @likeAZ
likeAZ
Retrait de consultantName de la table customerCard

### CODE REVIEW

File: src/GfiBundle/Entity/Comment.php [hunk 1]
```diff
@@ -43,9 +43,9 @@ class Comment
     private $date;
 
     /**
-     * @ORM\OneToOne(targetEntity="CustomerCard", mappedBy="idComment")
+     * @ORM\ManyToOne(targetEntity="CustomerCard", inversedBy="card")
```
Comment (Glangouet) [inline src/GfiBundle/Entity/Comment.php]: a modifier, comments <- à renommer

File: src/GfiBundle/Entity/CustomerCard.php [hunk 2]
```diff
@@ -91,19 +84,19 @@ class CustomerCard
...
      * @ORM\JoinColumn(name="idContact", referencedColumnName="id")
      */
     private $idContact;
 
     /**
-     * @ORM\OneToOne(targetEntity="Status", inversedBy="statusCards")
+     * @ORM\ManyToOne(targetEntity="Status", inversedBy="statusCards")
      * @ORM\JoinColumn(name="idStatus", referencedColumnName="id")
      */
     private $idStatus;
```
Comment (Glangouet) [inline src/GfiBundle/Entity/CustomerCard.php]: idstate

File: src/GfiBundle/Entity/CustomerCard.php [hunk 3]
```diff
@@ -91,19 +84,19 @@ class CustomerCard
...
 
     /**
-     * @ORM\OneToOne(targetEntity="Comment", inversedBy="commentCards")
+     * @ORM\OneToMany(targetEntity="Comment", mappedBy="idComment")
      * @ORM\JoinColumn(name="idComment", referencedColumnName="id")
      */
     private $idComment;
```
Comment (Glangouet) [inline src/GfiBundle/Entity/CustomerCard.php]: comments

File: src/GfiBundle/Entity/CustomerCard.php [hunk 4]
```diff
@@ -91,19 +84,19 @@ class CustomerCard
...
     /**
-     * @ORM\ManyToOne(targetEntity="ContactCustomer", inversedBy="contactCards")
+     * @ORM\OneToOne(targetEntity="ContactCustomer", inversedBy="contactCards")
      * @ORM\JoinColumn(name="idContact", referencedColumnName="id")
      */
     private $idContact;
 
     /**
-     * @ORM\OneToOne(targetEntity="Status", inversedBy="statusCards")
+     * @ORM\ManyToOne(targetEntity="Status", inversedBy="statusCards")
```
Comment (Glangouet) [inline src/GfiBundle/Entity/CustomerCard.php]: one to many

File: src/GfiBundle/Entity/CustomerCard.php (hunk 3 above)
Comment (Glangouet) [inline src/GfiBundle/Entity/CustomerCard.php]: comments

Comment (Glangouet) [reply src/GfiBundle/Entity/CustomerCard.php]: many to one

File: src/GfiBundle/Entity/CustomerCard.php [hunk 5]
```diff
@@ -288,30 +281,6 @@ public function getRate()
...
-     * Get consultantName
-     *
-     * @return string
-     */
-    public function getConsultantName()
-    {
-        return $this->consultantName;
-    }
-
-    /**
```
Comment (Glangouet) [inline src/GfiBundle/Entity/CustomerCard.php:314]: /**

File: src/GfiBundle/Entity/Status.php [hunk 6]
```diff
@@ -36,7 +36,7 @@ class Status
     private $date;
 
     /**
-     * @ORM\OneToOne(targetEntity="CustomerCard", mappedBy="idStatus")
+     * @ORM\OneToMany(targetEntity="CustomerCard", mappedBy="idStatus")
```
Comment (Glangouet) [inline src/GfiBundle/Entity/Status.php]: many to one

Comment (Glangouet) [inline src/GfiBundle/Entity/Status.php]: states

File: src/GfiBundle/Entity/Comment.php (hunk 1 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/Comment.php]: ok

File: src/GfiBundle/Entity/CustomerCard.php (hunk 4 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/CustomerCard.php]: ok

File: src/GfiBundle/Entity/CustomerCard.php (hunk 2 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/CustomerCard.php]: ok

File: src/GfiBundle/Entity/CustomerCard.php (hunk 3 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/CustomerCard.php]: ok

Comment (likeAZ) [reply src/GfiBundle/Entity/CustomerCard.php]: ok

File: src/GfiBundle/Entity/CustomerCard.php (hunk 5 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/CustomerCard.php:314]: ok

File: src/GfiBundle/Entity/Status.php (hunk 6 above)
Comment (likeAZ) [reply src/GfiBundle/Entity/Status.php]: ok

Comment (likeAZ) [reply src/GfiBundle/Entity/Status.php]: ok

**Pull request merged**